### PR TITLE
docs: adapt cookbooks to new Attachment logic

### PR DESCRIPTION
**Pull request by Dam-Buty**

### CODE REVIEW

File: typescript/dall-e-image-generation/package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@literalai/client": "^0.0.509",
+    "@literalai/client": "file:../../../cl-typescript-client/literalai-client-0.0.510.tgz",
```
Comment (desaxce) [inline typescript/dall-e-image-generation/package.json:13]: I guess we put it back to 0.0.510 before merging.

File: typescript/speech-to-emoji/with-literal/package.json [hunk 2]
```diff
@@ -14,7 +14,7 @@
     "docker:run": "docker run --env-file .env.docker -p 3001:3000 dambuty/emojifier"
   },
   "dependencies": {
-    "@literalai/client": "^0.0.509",
+    "@literalai/client": "file:../../../../cl-typescript-client/literalai-client-0.0.510.tgz",
```
Comment (desaxce) [inline typescript/speech-to-emoji/with-literal/package.json:17]: Same here.

File: typescript/dall-e-image-generation/index.ts [hunk 3]
```diff
@@ -36,21 +36,13 @@ const generateImage = async (prompt: string) => {
...
-          threadId: literalClient.getCurrentThread().id,
-        });
-
-      const attachment = new Attachment({
+      const { url: literalAiUrl } = await literalClient.api.createAttachment({
         name: "result",
-        objectKey,
+        path: localPath,
         mime: "image/png",
+        threadId: literalClient.getCurrentThread().id,
```
Comment (desaxce) [inline typescript/dall-e-image-generation/index.ts]: Since we're context-aware, I'd remove `threadId`.

Comment (desaxce) [reply typescript/dall-e-image-generation/index.ts]: Actually, we only create a run here, I don't think it makes sense to pass a thread.

File: typescript/speech-to-emoji/README.md [hunk 4]
```diff
@@ -122,38 +122,30 @@ To facilitate its use, we will generate the Run ID from the frontend using `cryp
...
-      input: { content: "Audio file" },
-      attachments: [attachment],
-    },
-  })
-  .send();
+  .wrap(async () => {
+    // Upload the file to Literal and add it as an attachment
+    const attachment = await literalClient.api.createAttachment({
+      content: formAudio,
+      threadId: literalClient.getCurrentThread().id,
```
Comment (desaxce) [inline typescript/speech-to-emoji/README.md:131]: Here, the threadId will be guessed by your wrapper logic.

**Review by desaxce — APPROVED** [review]

File: typescript/dall-e-image-generation/index.ts (hunk 3 above)
Comment (Dam-Buty) [reply typescript/dall-e-image-generation/index.ts]: done

File: typescript/dall-e-image-generation/package.json (hunk 1 above)
Comment (Dam-Buty) [reply typescript/dall-e-image-generation/package.json:13]: yup just need to release first, that's why i'm keeping this one as a draft

File: typescript/speech-to-emoji/README.md (hunk 4 above)
Comment (Dam-Buty) [reply typescript/speech-to-emoji/README.md:131]: you're right ! done

**Review by desaxce — APPROVED** [review]

**Pull request merged**